scripts/tests/docker-build/integration.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
+set -eu
+
+echo "running tests with $earthly"
+
+earthly=${earthly:=earthly}
+earthly=$(realpath "$earthly")
+frontend="${frontend:-$(which docker)}"
+tag=test-image:v1.2.3
+testdir=/tmp/earthtest
```

Comment:
I would make this more specific to this particular test.
```suggestion
testdir=/tmp/earthly-docker-build-test
```